model/project_ref.go

```diff
@@ -1360,6 +1365,22 @@ func saveProjectRefForSection(projectId string, p *ProjectRef, section ProjectRe
 					projectRefTriggersKey: p.Triggers,
 				},
 			})
+	case ProjectRefBuildBaronSection:
```

Comment:
This probably isn't right--this function is used for updating the new UI page and we haven't decided where Build baron / task annotation are going to live on this new page. I think we should just add a TODO with the ticket number for designing this.